backend/config/routes.rb

```diff
@@ -39,6 +40,8 @@
 
       resources :primary_donors
 
+      resources :redemptions, only: %i[create]
```

Comment:
This is never used right since we expose the route in coupons.

Reply:
Removed the route in coupons in https://github.com/Giving-Coupons/giving-coupons/pull/373/commits/69976800aa6060b035542a0b329e6d787bebf4c3